dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -322,3 +322,7 @@ CWPKI2070W.useraction=To avoid this warning message, set the certCheckerSchedule
 CWPKI2071W=CWPKI2071W: The certCheckerErrorSchedule property was set to {0}, which is shorter than the minimum schedule time. The certCheckerErrorSchedule property is reset to {1}.
 CWPKI2071W.explanation=The value for the certCheckerErrorSchedule property was below the minimum duration to check for expiring or revoked certificates and is reset to the minimum schedule time.
 CWPKI2071W.useraction=To avoid this warning message, set the certCheckerErrorSchedule property in the server configuration to a duration that is longer than the minimum schedule time. To use the default setting, remove the certCheckerErrorSchedule property from the server configuration.
+
+CWPKI2072W=CWPKI2072W: The ACME service could not read or write the historical ACME file at {0}. The error is: ''{1}''.
```

Comment:
For consistency with the other message I read in this file, I'd remove the colon from "The error is: ''{1}''."